chia/wallet/trading/trade_store.py

```diff
@@ -140,12 +172,21 @@ async def create(
             except aiosqlite.OperationalError:
                 pass  # ignore what is likely Duplicate column error
 
+            cursor = await conn.execute(
+                "SELECT name FROM sqlite_master WHERE type='table' AND name='MIGRATED_VALID_TIMES_TRADES'"
+            )
+            if await cursor.fetchone() is None:
+                needs_valid_times_migration = True
+                await conn.execute("CREATE TABLE MIGRATED_VALID_TIMES_TRADES(_ blob)")
```

Comment:
Why not just alter the table and add a new column with a default value? Therefore you don't have to all these migration things.

Reply:
This table being created is really just the equivalent of a boolean, unfortunately sql doesn't allow you to just store table metadata so I'm left with this hack.

The reason I have to do the migration is because we store the entire trade record in the DB as a blob under `trade_record`.  In the various DB methods, we will deserialize this blob using the `TradeRecord` streamable class and streamable is very specific that it can only deserialize things in exactly one way.  Therefore, I have to go into the DB and replace all of the blobs with blobs that are deserializable by the new `TradeRecord` class.